parsers/verilog.c

```diff
@@ -1603,6 +1603,23 @@ static int skipDelay(tokenInfo* token, int c)
 	return c;
 }
 
+static int skipClockEvent(tokenInfo* token, int c)
+{
+	if (c == '@')
+	{
+		c = skipWhite (vGetc ());
+		if (c == ('@')) {	// coverage_event: @@(block_event_expression)
```

Comment:
Is the parenthesis  around '@' needed?

Reply:
Off course, No.
But I found this condition is never covered and removed the if-clause.